src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountSevenDays = 50;
+  const discountThreeDays = 20;
+  const rentCar = 40 * days;
```

Comment:
Not clear variable name
```suggestion
  const basicPrice = 40 * days;
```